src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -100,6 +100,7 @@ public static IHostBuilder AddScriptHost(this IHostBuilder builder, Action<Scrip
                 {
                     configBuilder.Add(new HostJsonFileConfigurationSource(applicationOptions, SystemEnvironment.Instance, loggerFactory, metricsLogger));
                 }
+                configBuilder.Add(new FunctionsHostingConfigSource(SystemEnvironment.Instance));
```

Comment:
this is the new config source which leads to the test failure. Will need to update the test. Also will these config source file be available in v3?

And finally - not sure what this looks like in v4, but should we consider this a "Host" configuration? ie, we register it in `ConfigureHostConfiguration` - this way these values are available directly during `ConfigureAppConfiguration` via `HostContext.Configuration`